### DISCUSSION TITLE
I do someting wrong in my sensors.yaml file

### DISCUSSION THREAD

**Discussion by videologic2** [ESPresense/ESPresense#1209] [Support]
Hey!

I'm a beginner and trying to get the code in sensors.yaml good.  This is my output from mqtt explorer and my sensors.yaml file.  I hope for  detailed explaintion for a newbie;-)  
![sensorsyaml](https://github.com/ESPresense/ESPresense/assets/143906675/b0a79b05-ad4a-43df-bc4c-df691fddf917)
![MQTT explorer](https://github.com/ESPresense/ESPresense/assets/143906675/3519c8a3-2581-4327-84b2-039fa4b39d94)

Thanks!!  Daan

**Comment by videologic2** [comment]
Can anybody help? Thanks ;-)

**Comment by semontgo** [comment]
I hope someone who ACTUALLY knows what they are talking about responds because I've been at this for about 2 days.  But in the meantime,

Did you try 
Since you're setting up a device (sensor), not a room, I think you need 
     /espresense/**devices**/ 
instead of 
     /espresense/**rooms**/

I used Enroll on the ESPresense "devices" section to give my phone a name  You can find devices using your node's IP address:
http://192.168.0.xxx/ui/#/devices

Once I did that, I could use that name in the 
- platform: mqtt_room
  device_id: "**sarah_iphone**"
  name: "Sarah iPhone"
  state_topic: "espresense/devices/**sarah_iphone**"
  timeout: 5
  away_timeout: 120

**Reply by videologic2** [reply]
Wooww!!  That dit the trick. I was working like 2 days at this but with this the a finally can see a distcance. Thanks do much!
